### ISSUE TITLE
Loss is always Zero while creating a NER model

### ISSUE THREAD

**Issue by santhoshsridhar0304** [explosion/spaCy#3166]
**I'm creating a custom ner model using the CLI command**
`python -m spacy train en ./docs/model ./docs/data/train.json ./docs/data/sample_test_data.json -n 5 -P -T`.

**The problem is:**
The loss is always zero even though the train and test dataset are different. And obviously, the NER model is not performing as expected, even period (.) is considered as an entity.

### Output:

**dropout_from = 0.2 by default
dropout_to = 0.2 by default
dropout_decay = 0.0 by default
batch_from = 1 by default
batch_to = 16 by default
batch_compound = 1.001 by default
max_doc_len = 5000 by default
beam_width = 1 by default
beam_density = 0.0 by default
Warning: Unnamed vectors -- this won't allow multiple vectors models to be loaded. (Shape: (0, 0))
learn_rate = 0.001 by default
optimizer_B1 = 0.9 by default
optimizer_B2 = 0.999 by default
optimizer_eps = 1e-08 by default
L2_penalty = 1e-06 by default
grad_norm_clip = 1.0 by default
parser_hidden_depth = 1 by default
parser_maxout_pieces = 2 by default
token_vector_width = 128 by default
hidden_width = 200 by default
embed_size = 7000 by default
history_feats = 0 by default
history_width = 0 by default
Itn.  Dep Loss  NER Loss  UAS     NER P.  NER R.  NER F.  Tag %   Token %  CPU WPS  GPU WPS
0     0.000     0.000     0.000   5.882   40.000  10.256  100.000 100.000  349.7    0.0
1     0.000     0.000     0.000   13.333  40.000  20.000  100.000 100.000  781.1    0.0
2     0.000     0.000     0.000   0.000   0.000   0.000   100.000 100.000  851.0    0.0
3     0.000     0.000     0.000   5.000   40.000  8.889   100.000 100.000  705.4    0.0
4     0.000     0.000     0.000   0.000   0.000   0.000   100.000 100.000  717.5    0.0
Saving model...**

The training data set file size is 3.5 Mb.

Not able to find where I'm making a mistake.

## Environment
* Operating System: Windows 8.1
* Python Version Used: 3.6.8
* spaCy Version Used: 2.0.18

**Comment by santhoshsridhar0304** [comment]
Solved the issue. The loss was zero because my training file size is greater than the **max_doc_len = 5000** parameter. Once the parameter is modified the problem was solved.

**Note: Please make sure you change the _max_doc_length_ parameter if your training data is large.**
